app/src/main/java/com/kickstarter/mock/services/MockApolloClient.kt

```diff
@@ -43,12 +43,14 @@ import com.kickstarter.services.apiresponses.DiscoverEnvelope
 import com.kickstarter.services.apiresponses.ShippingRulesEnvelope
 import com.kickstarter.services.apiresponses.commentresponse.CommentEnvelope
 import com.kickstarter.services.apiresponses.updatesresponse.UpdatesGraphQlEnvelope
+import com.kickstarter.services.mutations.CreateAddressData
 import com.kickstarter.services.mutations.CreateBackingData
 import com.kickstarter.services.mutations.CreateCheckoutData
 import com.kickstarter.services.mutations.PostCommentData
 import com.kickstarter.services.mutations.SavePaymentMethodData
 import com.kickstarter.services.mutations.UpdateBackingData
 import com.kickstarter.viewmodels.usecases.TPEventInputData
+import com.stripe.android.paymentsheet.addresselement.AddressDetails
```

Comment:
nitpick for the future implementation, it is perfectly ok for the POC no need to change anythin, but we should try to define our own dataModel on the real implementation, instead using Stripes's defined one. If we ever want to substitute the 3rd party, or adopt for example Google Place and build the address collection piece ourself we should not be tied to an external data model.

Reply:
I think this is an unused import that i forgot to delete, as i created a datamodel for this `CreateAddressData`, totally agree that we should use our own over the third party implementation!